legacy/firmware/Makefile

```diff
@@ -19,11 +19,12 @@ ZKP_CFLAGS = \
 	-DUSE_EXTERNAL_ASM \
 	-DUSE_EXTERNAL_DEFAULT_CALLBACKS \
 	-DECMULT_GEN_PREC_BITS=4 \
-	-DECMULT_WINDOW_SIZE=8 \
+	-DECMULT_WINDOW_SIZE=2 \
```

Comment:
Why not leave the window size at 8 in the legacy build? We have no issues with lack of flash memory on T1, and we probably never will, because there are no major new features being added to T1 anymore.

We can also set `USE_PRECOMPUTED_CP=1` for the legacy build.